### PR TITLE
PMM-7278 Updated MongoDB TLS doc for auth mechanism

### PR DESCRIPTION
**Pull request by percona-csalguero**

### CODE REVIEW

File: docs/setting-up/client/mongodb.md [hunk 1]
```diff
@@ -180,13 +180,18 @@ pmm-admin add mongodb --tls \
...
+--authentication-mechanism=AUTHENTICATION-MECHANISM
+--authentication-database=AUTHENTICATION-DATABASE
 ```
 
 where:
 
 - `PATHTOCERT`: Path to TLS certificate file.
 - `IFPASSWORDTOCERTISSET`: Password for TLS certificate file.
 - `PATHTOCACERT`: Path to certificate authority file.
+- `AUTHENTICATION-MECHANISM`: Authentication mechanism. Default is empty. Use MONGODB-X509 for ssl certificates.
```
Comment (PaulJacobs-percona) [inline docs/setting-up/client/mongodb.md:192]:
```suggestion
- `AUTHENTICATION-MECHANISM`: Authentication mechanism. Default is empty. Use `MONGODB-X509` for SSL certificates.
```

Comment (PaulJacobs-percona) [reply docs/setting-up/client/mongodb.md:192]: @percona-csalguero are there any other possible values here? Where or how would a user find out what they should use?

File: docs/setting-up/client/mongodb.md [hunk 2]
```diff
@@ -180,13 +180,18 @@ pmm-admin add mongodb --tls \
...
+- `AUTHENTICATION-DATABASE`: Authentication database. Default is empty. Use $external for ssl certificates.
```
Comment (PaulJacobs-percona) [inline docs/setting-up/client/mongodb.md:193]: Is `$external` a literal string or a variable expansion? Are there any issues with quoting? Can you give an example of use. Plus, as above, what other values are possible and where are they found?

Comment (PaulJacobs-percona) [reply docs/setting-up/client/mongodb.md:193]:
```suggestion
- `AUTHENTICATION-DATABASE`: Authentication database. Default is empty. Use `$external` for SSL certificates.
```

File: docs/setting-up/client/mongodb.md [hunk 3]
```diff
@@ -196,6 +201,7 @@ where:
 2. Look in the *Services* tab for a matching *Service Type* (MongoDB), *Service name*, *Addresses*, and any other values used when adding the service.
 3. Look in the *Agents* tab to check the desired data source is being used.
 4. If your MongoDB instance is configured to use TLS, click on the **Use TLS for database connection** check box and fill in TLS certificates and keys.
+    1. If you use TLS, the authentication mechaninsm is being set automatically to `MONGODB-X509`.
```
Comment (PaulJacobs-percona) [inline docs/setting-up/client/mongodb.md:204]:
```suggestion
    1. If you use TLS, the authentication mechanism is automatically set to `MONGODB-X509`.
```

**Review by PaulJacobs-percona — CHANGES_REQUESTED** [review]
See inline comments and questions.

File: docs/setting-up/client/mongodb.md (hunk 1 above)
Comment (percona-csalguero) [reply docs/setting-up/client/mongodb.md:192]: The default is `SCRAM` and `MONGODB-X509` is for X509 certs.
Other values depend on the server config: https://docs.mongodb.com/manual/reference/parameters/#mongodb-parameter-param.authenticationMechanisms

**Review by PaulJacobs-percona — APPROVED** [review]

**Pull request merged**